manifests/charts/ztunnel/templates/rbac.yaml

```diff
@@ -10,7 +10,11 @@ metadata:
   name: ztunnel
   namespace: {{ .Release.Namespace }}
   labels:
-    {{- .Values.labels | toYaml | nindent 4}}
+    istio.io/component: ztunnel
```

Comment:
Actually - if you are doing the exact same thing here, it should just be a template used in both places.


e.g. in `_helpers.tpl`:

```
{{- define "istio-labels" }}
    app.kubernetes.io/name: ztunnel
    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
    app.kubernetes.io/instance: {{ .Release.Name }}
{{- end }}
```

and then 

```
labels:
  {{ template "istio-labels" . }}
```

here and in the other spot.

Reply:
`app.kubernetes.io/managed-by: Helm` is automatically added I think. It's the only label I see on zt when I install with helm.

Reply:
It's added if you `helm install` but not if you `helm template` (which makes sense).

Reply:
The chart label would not work from within the _helper.tpl file. I don't really understand why. I am not going to fight it any more so I left it out. We can circle back later if it's important.

Reply:
I think it's a matter of escaping YAML properly for `sprig`, but once the template is in place it becomes less of an issue to solve this centrally + consistently later, so that's fine with me for now.